<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestApiPractiveClass</class>
 <widget class="QMainWindow" name="RestApiPractiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RestApiPractive</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
      <item>
       <widget class="QLineEdit" name="lineEdit_url_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_get_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RestApiPractive.qrc"/>
 </resources>
 <connections/>
</ui>
